stdlib/universe/universe.flux

```diff
@@ -2,6 +2,9 @@ package universe
 
 import "system"
 import "date"
+import "math"
+import "strings"
+import "regexp"
```

Comment:
I see you took these packages out of the prelude in `compile.go`, curious about why?

Reply:
re: above